test/parallel/test-os.js

```diff
@@ -125,7 +125,7 @@ console.error(interfaces);
 switch (platform) {
   case 'linux':
     {
-      const filter = function(e) { return e.address === '127.0.0.1'; };
+      const filter = (e) => { return e.address === '127.0.0.1'; };
```

Comment:
Sure! It's the predominant style and if it inches us closer to being able to enable `arrow-body-style` in `.eslintrc.yaml`, then I'm all for it.